docs/src/test.md

```diff
@@ -46,6 +46,93 @@ pytest -svv forge/test/mlir/test_ops.py::test_add
 
 > - The `-svv` flag is optional and used to display more information about the test run.
 
+## Pytest practices
```

Comment:
Looks good! 

What do you think about moving it to as sub-section of test.md file? E.g. creating a new file, and nesting it under Testing section. 
<img width="206" alt="image" src="https://github.com/user-attachments/assets/93974433-e7cf-4069-9eeb-c01c47ed1e2d" />

E.g.
```
...
4. Testing
    4.1. Pytest
...
```